packages/expo-image/ios/EXImageView.m

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020-present 650 Industries. All rights reserved.
+
+#import <expo-image/EXImageView.h>
+#import <React/RCTConvert.h>
+
+@implementation EXImageView
+
+- (void)setSource:(NSDictionary *)source
+{
+  NSURL *imageUrl = [RCTConvert NSURL:source[@"uri"]];
```

Comment:
Perhaps we should add a static NSString for referencing the @"uri" key here, similar to the java code.